apps/api/src/domain/models/term/entities/term.entity.ts

```diff
@@ -91,15 +96,12 @@ export class Term extends Resource {
     })
     readonly contributorId?: AggregateId;
 
-    /**
-     * TODO Make this a mediaItemId
-     */
     @NonEmptyString({
         isOptional,
-        label: 'Audio Filename',
-        description: 'a url link to the audio file',
+        label: 'Audio Item ID',
+        description: 'a system reference to the audio file',
     })
-    readonly audioFilename?: string;
+    readonly audioItemId?: string;
```

Comment:
Did this really have no consequences on tests? I suppose this is only set in the present test and is not set in either of the create commands.